client/src/components/App/Content/Content.tsx

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
-import {Route, Switch, Redirect } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { Route, Switch, Redirect } from 'react-router-dom';
 
+import UserType from 'models/enums/UserType';
+import StoreStateType from 'redux/storeStateType';
 import LoadingSpinner from 'commons/LoadingSpinner/LoadingSpinner';
-import { investigationFormRoute, landingPageRoute, usersManagementRoute, indexRoute } from 'Utils/Routes/Routes';
+import { investigationFormRoute, landingPageRoute, usersManagementRoute, indexRoute, adminLandingPageRoute } from 'Utils/Routes/Routes';
 
 import SignUp from './SignUp/SignUp';
-import LandingPage from  './LandingPage/LandingPage';
+import UsersManagement from './UsersManagement/UsersManagement';
 import InvestigationForm from './InvestigationForm/InvestigationForm';
-import UsersManagement from './UsersManagement/UsersManagement'
+import LandingPage from './LandingPage/defaultLandingPage/LandingPage';
+import adminLandingPage from './LandingPage/adminLandingPage/adminLandingPage';
 
 const Content: React.FC<Props> = ({ isSignUpOpen, handleSaveUser, handleCloseSignUp }): JSX.Element => {
-  
+    const userType = useSelector<StoreStateType, number>(state => state.user.data.userType);
+
     return (
         <>
             <Switch>
                 <Route path={investigationFormRoute} component={InvestigationForm} />
                 <Route path={landingPageRoute} component={LandingPage} />
+                <Route path={adminLandingPageRoute} component={adminLandingPage} />
                 <Route path={usersManagementRoute} component={UsersManagement} />
-                <Redirect from={indexRoute} to={landingPageRoute}/>
+                {
+                    (userType === UserType.ADMIN || userType === UserType.SUPER_ADMIN) ?
```

Comment:
Once the landing page opens and once the table, this is probably an asynchronous problem on the userType